opsramp/base.py

```diff
@@ -146,10 +203,18 @@ def process_result(url, resp):
                 resp.content
             )
             raise RuntimeError(msg)
-        try:
-            return resp.json()
-        except: # noqa
-            return resp.text
+        #try:
+        data = resp.json()
```

Comment:
is the try obsolete now?

Reply:
No. That's an old-fashioned "forgot to undo".